File: back-end/src/main/java/pl/straburzynski/campfiresongs/config/AppConfig.java
package pl.straburzynski.campfiresongs.config;


import lombok.Data;
import org.springframework.beans.factory.annotation.Value;
import org.springframework.context.annotation.Configuration;

@Configuration
@Data
public class AppConfig {

    @Value("${app.externalApiUrl.spiewnikWywrota}")
    private String externalApiUrlSpiewnikWywrota;

    @Value("${app.externalApiUrl.ultimateGuitar}")
    private String externalApiUrlUltimateGuitar;

}

File: back-end-kotlin/src/main/kotlin/pl/straburzynski/campfiresongs/config/AppConfig.kt
package pl.straburzynski.campfiresongs.config

import org.springframework.beans.factory.annotation.Value
import org.springframework.context.annotation.Configuration

@Configuration
class AppConfig {
    @Value("\${app.externalApiUrl.spiewnikWywrota}")
    lateinit var externalApiUrlSpiewnikWywrota: String

    @Value("\${app.externalApiUrl.ultimateGuitar}")
    lateinit var externalApiUrlUltimateGuitar: String

}